numpyro/distributions/directional.py

```diff
@@ -400,7 +400,7 @@ def norm_const(self):
 
         fs = (
             lbinoms.reshape(-1, 1)
-            + 2 * m * jnp.log(corr)
+            + m * jnp.log(corr**2)
             - m * jnp.log(4 * jnp.prod(conc, axis=-1))
         )
         fs += log_I1(49, conc, terms=51).sum(-1)
```

Comment:
I'm not sure that I follow but it seems to me that norm_const does not have batch dimensions after fs.max() operator below. Is it intended?

Reply:
Probably you can remove mfs term because logsumexp does it under the hood.

Reply:
I see. I'll take out the `mfs` term.